examples/python/D0ToKs0KpKm.xml

```diff
@@ -1,417 +1,418 @@
 <?xml version="1.0" encoding="utf-8"?>
-
```

Comment:
why are these two xml files here in the first place. These are left overs from a test verifying the angular distribution

Reply:
True. But I don't want to address those (indeed malfunctioning) examples in this PR. An issue can be made for this

Reply:
but cant this be just deleted instead of reformatting the whole file and an deleting it afterwards again?

Reply:
if you want this pr to be clean then I would suggest to apply a separate chore PR before that

Reply:
In that case we'd better completely remove those examples

Reply:
and I'm not sure if we really want that, that's why I say, better make an issue for this

Reply:
Will be rebased onto #101